src/main.rs

```diff
@@ -210,46 +236,48 @@ async fn main() -> Result<(), Box<dyn Error>> {
                 }
                 _ => {}
             },
-            InputEvent::Mouse(MouseEvent::Up(MouseButton::Left, col, row, _)) => {
-                match target_areas.read().iter().rev().find(|(_, area)| {
+            InputEvent::Mouse(MouseEvent::Up(MouseButton::Left, col, row, _)) => match target_areas
+                .read()
+                .iter()
+                .rev()
+                .find(|(_, area)| {
                     area.left() <= col
                         && area.right() >= col
                         && area.top() <= row
                         && area.bottom() >= row
                 }) {
-                    Some((UiTarget::StockSymbol, _)) | Some((UiTarget::StockName, _)) => {
-                        stock_symbol_input_state.active = !stock_symbol_input_state.active;
-                        time_frame_menu_state.active = false;
-                    }
-                    Some((UiTarget::TimeFrame, _)) => {
-                        time_frame_menu_state.active = !time_frame_menu_state.active;
-                        stock_symbol_input_state.active = false;
+                Some((UiTarget::StockSymbol, _)) | Some((UiTarget::StockName, _)) => {
```

Comment:
Omg this indentation is so weird... :cry:

Reply:
Is this a change in `rustfmt`? Hmm... I'm using stable Rust.

Reply:
Me `stable 1.43.1`, let me check again.